pkg/util/platform.go

```diff
@@ -0,0 +1,30 @@
+package util
+
+import (
+	"fmt"
+
+	"github.com/containerd/containerd/platforms"
+	v1 "github.com/opencontainers/image-spec/specs-go/v1"
+)
+
+func normalizeContainerPlatfrom(platform v1.Platform) v1.Platform {
+	platform.OS = "linux"
+	return platforms.Normalize(platform)
+}
```

Comment:
why setting this to linux?

Reply:
All docker runtimes run images of `linux/amd64` or `linux/arm64` architecture -- even on `darwin` or `windows`.

Reply:
probably worth adding this in as a comment to the code